test/AutoRest.TestServer.Tests/custom-baseURL.cs

```diff
@@ -18,5 +19,11 @@ public class custom_baseURL : TestServerTestBase
         [Test]
         public Task CustomBaseUriMoreOptions() => TestStatus(async (host, pipeline) =>
             await new custom_baseUrl_more_options.PathsClient(ClientDiagnostics, pipeline, "test12", dnsSuffix: host.ToString()).GetEmptyAsync( string.Empty, string.Empty, "key1",  "v1"));
+
+        [Test]
+        public void ThrowsIfHostIsNull()
+        {
+            Assert.Throws<ArgumentNullException>(() => new custom_baseUrl.PathsClient(ClientDiagnostics, null , null));
```

Comment:
Which parameter is being checked for null here?

Reply:
Last parameter i.e. `host`. Replaced other param with random value to make this clear.